examples/instance_segmentation/eval_sbd.py

```diff
@@ -14,9 +14,9 @@
 def main():
     parser = argparse.ArgumentParser()
     parser.add_argument(
-        '--model', choices=('fcis_resnet101'),
+        '--model', choices=['fcis_resnet101'],
         default='fcis_resnet101')
-    parser.add_argument('--pretrained_model')
```

Comment:
This was correct because it is consistent with other examples.

Reply:
how about changing all others?

Reply:
I agree with @knorth55. `--foo-bar` style is more common.

Reply:
OK

Reply:
move to #616